### PR TITLE
Fix/bluetooth

### PR DESCRIPTION
**Pull request by MeriyemBilgili**
### Checklist

- [x] Tests have been written/updated (if necessary)
  - [x] All tests are passing
- [x] Docs have been updated (if necessary)
- [x] Updated `Linear` issue to `In Review`

### Description

The issue with turning on Bluetooth has been resolved. To enable Bluetooth, you need to click on the 'Enable Bluetooth' button. You will then be redirected to the Bluetooth page on your mobile phone to enable Bluetooth there.

### CODE REVIEW

File: apps/expo/src/lang/nl-NL.json [hunk 1]
```diff
@@ -152,15 +152,22 @@
           }
         },
         "bluetooth": {
-          "enabling": "Bluetooth inschakelen...",
+          "enabling": "Bluetooth is momenteel uitgeschakeld. Om verder te kunnen gaan, dient u uw Bluetooth in te schakelen",
           "alert": {
             "title": "Bluetooth toestemming",
             "message": "Deze app gebruikt Bluetooth, maar registreert uw locatie niet. Op Android valt Bluetooth onder locatiediensten. Daarom krijgt u zo een vraag over locatie.",
```
Comment (Labhatorian) [inline apps/expo/src/lang/nl-NL.json:158]: Should mention nearby devices permission too

File: apps/expo/src/lang/en-US.json [hunk 2]
```diff
@@ -152,15 +152,23 @@
           }
         },
         "bluetooth": {
-          "enabling": "Enabling Bluetooth...",
+          "enabling": "Bluetooth is currently disabled. To proceed, please enable your Bluetooth.",
           "alert": {
             "title": "Bluetooth permission",
             "message": "This app uses Bluetooth, but does not register your location. On Android devices, Bluetooth is considered part of location services. That's why you will get a question about location.",
```
Comment (Labhatorian) [inline apps/expo/src/lang/en-US.json:158]: Should mention nearby devices permission too

**Review by Labhatorian — CHANGES_REQUESTED** [review]
Need to make sure people know to add nearby devices permission too. Otherwise, it seems to work fine when I test it. No crashes and I get told which permissions need to be added + enable bluetooth.

Btw, you don't need to remake a PR for this. new commits will get added

**Review by Labhatorian — APPROVED** [review]
I've tested it and it works for me

File: apps/expo/src/lang/nl-NL.json [hunk 3]
```diff
@@ -152,15 +152,22 @@
           }
         },
         "bluetooth": {
-          "enabling": "Bluetooth inschakelen...",
+          "enabling": "Bluetooth is momenteel uitgeschakeld. Om verder te kunnen gaan, dient u uw Bluetooth in te schakelen",
           "alert": {
             "title": "Bluetooth toestemming",
-            "message": "Deze app gebruikt Bluetooth, maar registreert uw locatie niet. Op Android valt Bluetooth onder locatiediensten. Daarom krijgt u zo een vraag over locatie.",
-            "button": "Oké, vraag maar!"
+            "message": "Deze app maakt gebruik van Bluetooth, maar registreert uw locatie niet. Op Android valt Bluetooth onder locatiediensten. Daarom zult u binnenkort een vraag krijgen over locatietoegang. Bovendien dient u toestemming te geven aan apparaten in de buurt en locatie.",
```
Comment (Labhatorian) [inline apps/expo/src/lang/nl-NL.json:158]: This line shows on iOS too.
iOS only needs Bluetooth and Location and we probably should not mention Android.

File: apps/expo/src/screens/home/SearchDeviceScreen.tsx [hunk 4]
```diff
@@ -71,15 +71,15 @@ export default function SearchDeviceScreen({ navigation, route }: SearchDeviceSc
     Alert.alert("Error", err, [
       Platform.OS === "ios"
```
Comment (Labhatorian) [inline apps/expo/src/screens/home/SearchDeviceScreen.tsx:72]: Reuse this to change the line I mentioned

**Review by Labhatorian — CHANGES_REQUESTED** [review]
Just tested it on iOS. It behaves differently but not in a bad way, I believe the main non-happy flow was on Android.
iOS is good enough to help people understand they need Bluetooth turned on.

**Review by Labhatorian — APPROVED** [review]
Should now be fixed for iOS too Since I worked on it too, I'll approve it but someone else should still check.

**Review by ThomasBuh — APPROVED** [review]

File: apps/expo/src/screens/home/SearchDeviceScreen.tsx [hunk 5]
```diff
@@ -120,6 +129,7 @@ export default function SearchDeviceScreen({ navigation, route }: SearchDeviceSc
     setIsError(false);
 
     const hasBluetoothPermission = await checkBluetoothPermission();
+    console.log(hasBluetoothPermission);
```
Comment (Niels995) [inline apps/expo/src/screens/home/SearchDeviceScreen.tsx:132]: Dont leave console.log in the final code...

**Review by Niels995 — APPROVED** [review]

**Pull request merged**